test/spec/modules/vidazooBidAdapter_spec.js

```diff
@@ -79,7 +79,7 @@ const REQUEST = {
   }
 };
 
-describe('VidazooBidAdapter', function () {
+describe.only('VidazooBidAdapter', function () {
```

Comment:
This is running only this test file. I think you forgot to remove `.only`?

Reply:
yep! they had been giving me some errors so i was testing locally just them, forgot to remove before committing...will fix that now